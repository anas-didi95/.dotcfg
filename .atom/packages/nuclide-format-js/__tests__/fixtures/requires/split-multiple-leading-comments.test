/**
 * Header comment
 */

/**
 * Function description.
 */
function foo() {
  one();
}
