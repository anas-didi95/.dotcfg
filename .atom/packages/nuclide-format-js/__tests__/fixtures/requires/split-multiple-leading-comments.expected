/**
 * Header comment
 */

const one = require('one');

/**
 * Function description.
 */
function foo() {
  one();
}
